src/reducers/userReducer.js

```diff
@@ -0,0 +1,49 @@
+import * as ActionConstants from '../constants/actions';
+import store from 'react-native-simple-store';
+import * as R from 'ramda';
+
+const InitialUserState = { };
+
+export default function user(state=InitialUserState, action) {
+    switch (action.type) {
+        case ActionConstants.SYNC_USER_STORE: {
+            store.save('@zooniverse:user', { state })
+            return state;
+        }
+        case ActionConstants.SET_USER_FROM_STORE: {
+            if ( action.storeData !== undefined ) {
+                return action.storeData;
```

Comment:
Perhaps I am missing something here. Don't you need to make a copy of the state to retain the current key/values? `Object.assign()`.etc? I would think you would overwrite other items you're keeping track of in a line like this.

Reply:
So this is the way it was working before the refactor. Basically this actions purpose is to replace the current state with the one in persistent storage

Reply:
Ok. I suppose this makes sense as the state is referenced in each of the switch cases. It's just a different approach from what we've been doing on some of the other frontends, so I'm just used to that approach.